nishimweelysee/Elysee-CapstoneProject
# Ft delete article

#### What does this PR do?
> This PR used to delete an article either blog or portfolio. some article may not be liked by the users or not well written and the blogger need to delete it. he will use this delete button to remove that article.
>
#### Description of Task to be completed
> Delete Button that will help to delete.
>
#### How should this be manually tested?
> 1. Open a website
>2.  go to admin tab
>3.  then go to Blog or Portfolio Tab.
>4.  Then you will see a delete button you have to search an article you need to delete or find it on list then delete it.
#### Any background context you want to provide?
>
#### What are the relevant Trello story?
>FT(Delete an existing article) Use this link https://trello.com/c/eTF0fLNC to Visit my trello cards.
>
#### Screenshots (if appropriate)
>
> This is for deleting a blog
>
<img width="772" alt="deleteblog" src="https://user-images.githubusercontent.com/56052588/92359498-a1c14480-f0eb-11ea-8550-1be82c367290.PNG">
>This is for deleting a portfolio
>
<img width="762" alt="deleteportfolio" src="https://user-images.githubusercontent.com/56052588/92359499-a38b0800-f0eb-11ea-88a3-196739b55a58.PNG">
>
> Thank you!!
